test/runtests.jl

```diff
@@ -10,19 +10,19 @@ solvers = [
            ODE.ode2_midpoint,
            ODE.ode2_heun,
            ODE.ode4,
-           ODE.ode4ms,
-           ODE.ode5ms,
+           # Ode.ode4ms,
```

Comment:
No problem at all, I figured this was still in progress. I'm actually attending JuliaCon in Boston, if you are here also, it would be great to find some time and working on progressing this PR. Also, I ran some performance tests with @mauro3's IVPTestSuite.jl, and I could show you some of the results and how to run them yourself.